src/adagrams.js

```diff
@@ -1,8 +1,148 @@
+const alpha = [
+  "A", "A", "A", "A", "A", "A", "A", "A", "A",
+  "B", "B",
+  "C", "C",
+  "D", "D", "D", "D",
+  "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E",
+  "F", "F",
+  "G", "G", "G",
+  "H", "H",
+  "I", "I", "I", "I", "I", "I", "I", "I", "I",
+  "J",
+  "K",
+  "L", "L", "L", "L",
+  "M", "M",
+  "N", "N", "N", "N", "N", "N",
+  "O", "O", "O", "O", "O", "O", "O", "O",
+  "P", "P",
+  "Q",
+  "R", "R", "R", "R", "R", "R",
```

Comment:
This will definitely get you the right distribution, but it will be a bit awkward if anyone asks you to change it. Talk to your peers about how they dried this part up.